app/views/admin/shared/_nested_sidebar.erb

```diff
@@ -0,0 +1,18 @@
+<% menu_items.each do |group_name, group| %>
+  <% if group.length == 1 %>
+    <a class="crayons-link crayons-link--block"
+       href="/admin/<%= group[0][:controller] %>"
+    >
+      <%= group_name.to_s.gsub("_", " ").titleize %>
```

Comment:
non blocking: I wonder, since `AdminMenu` is essentially a view object, if this logic should be moved into `Menu.item()` into a `display_name` attribute of the hash built there

Reply:
I've kept the admin helper around for some of the  view logic in the admin helper. Going to pop this in there ✅